packages/labs/observers/src/resize-controller.ts

```diff
@@ -38,9 +38,12 @@ export interface ResizeControllerConfig<T = unknown> {
    */
   callback?: ResizeValueCallback<T>;
   /**
-   * By default the `callback` is called without changes when a target is
-   * observed. This is done to help manage initial state, but this
-   * setup step can be skipped by setting this to true.
+   * The browser's `ResizeObserver` triggers immediately when a target is first
+   * observed. This behavior may be undesirable in a Lit component, as it may
+   * cause an extraneous `updated()` lifecycle callback after the initial
+   * render. If the performance impact of this is a concern, set `skipInitial`
+   * to true, which will cause the `ResizeController` to automatically ignore
+   * the first callback from the `ResizeObserver`.
```

Comment:
I think it'll read better in hover-over docs to have a short positive description of the effect of the flag, followed by a longer description of why.

Also "`updated()` lifecycle callback" is too specific, the whole lifecycle would be run again.

```suggestion
   * When `true`, ignores the first `ResizeController` callback for each
   * observed element.
   *
   * The browser's `ResizeObserver` triggers immediately when a target is first
   * observed. This behavior may be undesirable in a Lit component, as it may
   * cause an extraneous update lifecycle iteration after the initial
   * render. If the performance impact of this is a concern, set `skipInitial`
   * to true, which will cause the `ResizeController` to automatically ignore
   * the first callback from the `ResizeObserver`.
```